README.md

```diff
@@ -1 +1,23 @@
-# parasoft-demo-app-mobile
\ No newline at end of file
+# parasoft-demo-app-mobile
+The Parasoft Demo Application is an example Android application. The application is configurable, and is used to demonstrate functionality in a variety of Parasoft tools.
+
+## Getting Started
+### Create emulator
+1. Open **Device Manager**, click **Create device**.
+2. Choose a device which has the resolution higher than **720x1280**, we recommend the device **Pixel 5 (1080x2340)**.
+3. Click **next**, choose an API version which **higher or equal to 32**, then click finish.
+
+### Build apk
```

Comment:
Most likely the user of this project will not open Android Studio, so we should provide steps to build from CMD
1. Move this section to top. 
2. Change title to be "Building unsigned APK".
3. Include step to run from command line - ./gradlew assembleDebug